docs/fundamentals/code-analysis/style-rules/naming-rules.md

```diff
@@ -19,31 +19,29 @@ helpviewer_keywords:
 ---
 # Naming rules
 
-Naming rules concern the naming of .NET programming language code elements, such as classes, properties, and methods. For example, you can specify that public members must be capitalized or that private fields must begin with `_`.
+In your `.editorconfig` file, you can define **naming rules** for how .NET programming language code elements&mdash;such as classes, properties, and methods&mdash;should be named. For example, you can specify that public members must be capitalized, or that private fields must begin with `_`.
 
-A naming rule has three parts:
+A naming rule has three components:
 
-* The group of symbols it applies to.
-* The naming style to associate with the rule.
+* The **symbol group** -- the group of symbols the rule applies to.
+* The **naming style** to associate with the rule.
 * The severity for enforcing the convention.
 
-You define naming rules in an EditorConfig file.
-
 ## General syntax
 
 To define a naming rule, symbol group, or naming style, set one or more properties using the following syntax:
 
 ```ini
-<prefix>.<title>.<propertyName> = <propertyValue>
+<kind>.<title>.<propertyName> = <propertyValue>
 ```
 
 Each property should only be set once, but some settings allow multiple, comma-separated values.
 
 The order of the properties is not important.
 
-### \<prefix>
+### \<kind>
 
-**\<prefix>** specifies which kind of element is being defined&mdash;naming rule, symbol group, or naming style&mdash;and must be one of the following:
+**\<kind>** specifies which kind of element is being defined&mdash;naming rule, symbol group, or naming style&mdash;and must be one of the following:
 
 | To set a property for | Use the prefix | Example |
```

Comment:
We should probably rename the column header for the second column. Maybe `Use \<kind> value`?

Reply:
Agreed.

Reply:
Done.